app/Commands/LocalDocker/Share.php

```diff
@@ -22,7 +22,7 @@ class Share extends BaseLocalDocker {
      *
      * @var string
      */
-    protected $signature = 'share';
+    protected $signature = 'share {directory=wp-content : The name of the wp-content folder, if renamed}';
```

Comment:
Should this be a named argument? Like `so share --content-dir={my content/directory/path}` JIC you add other arguments later?

Reply:
You're right, I'll update.

Reply:
@nickpelton updated, please have a look.